packages/create-astro/src/index.ts

```diff
@@ -284,10 +285,61 @@ export async function main() {
 		ora().info(dim(`--dry-run enabled, skipping.`));
 	} else if (gitResponse.git) {
 		await execaCommand('git init', { cwd });
+		ora().succeed('Git repository created!');
 	} else {
 		ora().info(dim(`Sounds good! You can come back and run ${cyan(`git init`)} later.`));
 	}
 
+	const tsResponse = await prompts(
+		{
+			type: 'select',
+			name: 'typescript',
+			message:
+				'Astro is a TypeScript-first framework. How strict would you like the typechecking to be?',
+			choices: [
+				{
+					title: 'Relaxed',
+					description: "Not a big fan of TypeScript? No issues, we'll keep this kind",
```

Comment:
```suggestion
					description: "Not a big fan of TypeScript? No worries, we'll keep things casual",
```
I LOVE this option! This is such user empathy!  😄 

I would suggest adjusting the message a little bit, though. When I read it, I read "kind" expecting "kind of..." So, I thought it was going to say, "We'll keep this kind of...." and then it stopped.  

Often designing messages for the widest audience possible, we would try to avoid slang etc. So, you can't go wrong with a very factual description here of what will/won't happen. But, I think if you want to convey this friendliness, there are still some choices that could work.  Maybe like....

First part:  I think "No worries" is much more common across an international audience than "No issues"  (issues might make people think of bugs? or is there an "issue" with this process?) I'd consider "worries" instead.

Second part: this is a little harder, but here are some possibilities that might be a bit more accessible while still being fun, maybe?

- we'll keep things casual
- we'll take it easy on you
- we won't make you use it / you don't have to use it
- we'll turn off the red squigglies
- we'll try to help, but won't insist
... ?

Reply:
I agree! I used `No issues` because `No worries` is used earlier in the steps and my brain immediately went in "Find synonyms" mode 😄

For the second part, we can't say that the user won't have to use it or that it'll disable anything because it effectively doesn't. TypeScript is always enabled, just with friendlier settings. `we'll keep things casual` seems good to me!

Reply:
Here were my suggestions, lmk what you think! https://github.com/withastro/astro/pull/4179#issuecomment-1207069870

Reply:
One thought on formatting: because this is the last question, what if the "don't love typescript" message came after "setup complete!" That way it wouldn't break up the series of prompts mid-way through.